scripts/coverage

```diff
@@ -266,9 +313,9 @@ class HtmlReport(Report):
 @dataclass
 class GithubPagesReport(HtmlReport):
     output_dir: Path
-    commit_url: str
+    commit_url: str = 'https://local/deadbeef'
```

Comment:
Probably unintentional

Reply:
No, in fact, I added this to provide a fallback for local rendering. This dummy url will make it to the report.